.ci/check_source_file_changes.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { xrun } from "../prettier/util.ts";
+import { red, green } from "../colors/mod.ts";
+
+/**
+ * Checks whether any source file is changed since the given start time.
+ * If some files are changed, this function exits with 1.
+ */
+async function main(startTime: number): Promise<void> {
```

Comment:
Promise<boolean>

Reply:
Why? This is one of script which should exit with code 1 if files aren't formatted.